components/link/link.js

```diff
@@ -44,13 +45,23 @@ export const linkStyles = css`
 			color: var(--d2l-color-ferrite);
 		}
 	}
```

Comment:
This CSS gets applied to native `<a class="d2l-link">` as well that consumers are using. Since they don't understand the "new window" stuff, any new-window-specific CSS can go down below in the CSS that's just used for the `<d2l-link>` web component.